src/Core/Plugin.php

```diff
@@ -241,8 +241,7 @@ public static function loadAll(array $options = [])
             }
         }
         if (Configure::check('plugins')) {
-            $plugins = array_merge($plugins, array_keys(Configure::read('plugins')));
-            $plugins = array_unique($plugins);
+            $plugins = array_unique(array_merge($plugins, array_keys(Configure::read('plugins'))));
```

Comment:
Same, and especially here.

Reply:
I too don't like this. Reduces code readability.

Reply:
Ok, I probably don't have enough context of the project. Will revert this specific inspection fixes.